lib/src/install.rs

```diff
@@ -859,6 +860,43 @@ pub(crate) fn setup_tmp_mounts() -> Result<()> {
     Ok(())
 }
 
+#[context("Ensuring sys mounts")]
+pub(crate) fn setup_sys_mounts() -> Result<()> {
```

Comment:
This makes sense as a name, though let's add some commentary here, something like:

```
/// By default, podman/docker etc. when passed `--privileged` mount `/sys` as read-only, but non-recursively.  We selectively grab sub-filesystems that we need.
```

I think your naming is right though because it'd make sense to also handle `selinuxfs` exactly this same way.  (Don't need to do it in this PR though)